install-instructions.md

```diff
@@ -102,7 +102,19 @@ You need [Java 17 or higher](https://adoptium.net/) to play the game.
 
 ### [Minecraft Launcher](https://www.minecraft.net/en-us/download) (vanilla)
 
-For macOS or Linux [you need Java](https://adoptium.net/) to run the Fabric Installer.
+The Vanilla Installer is currently in open beta. If you run into any issues, please report them on Discord and use the manual method.
+
+1. Download [vanilla installer](https://download.fo/vanilla) and run it
+    * Windows: Open the .exe file, click "More info" and then "Run anyway".
+    * macOS: Open the .dmg file, **right click** the "Fabulously Optimized Installer" and click "Open", click "Open" again when asked in a prompt.
+    * Linux: Open the .appimage file and click "Run once" when prompted.
+2. Select preferred FO/Minecraft version and install
+3. Run Minecraft Launcher. You'll see the new installation active, click PLAY.
+
+Note: you may now see a new file _paigaldaja_meta.json_ next to the installer. This is a configuration file that ensures smoother upgrades and remembers some of your install settings.
```

Comment:
this is technically incorrect here; `paigaldaja_meta.json` is saved in the profile dir